packages/portal/src/components/entity/EntityTable.vue

```diff
@@ -107,8 +172,10 @@
         let collections = response.data[this.cacheKey];
         if (this.type === 'organisations') {
           collections = collections.map(this.organisationData);
+          this.collections = collections; // Do not freeze as _showDetails prop needs to be reactive for toggling the details display on small screens
```

Comment:
Shouldn't this be for all entity types, not just organisations?

Reply:
Previously we freezed all the collections objects. Since we started using the row details for organisations on this PR,  we need the collection objects to be reactive. Under the hood the BoostrapVue row details functionality sets and toggles the `_showDetails` prop on each collection. At the moment, only the organisations type uses this functionality. So I we can keep freezing the collections for the other types. I'm assuming that's better for performance.